### PR TITLE
route index added

### PR DESCRIPTION
**Pull request by biswasarpita**

### CODE REVIEW

File: Routes/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.delete('/', (req, res) => {
```
Comment (amitroy) [inline Routes/index.js]: channge to router.get

File: Routes/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    res.setEncoding('hello');
```
Comment (amitroy) [inline Routes/index.js]: res.send instead

**Review by amitroy — COMMENTED** [review]
Kindly fix the comments

File: Routes/index.js (hunk 1 above)
Comment (biswasarpita) [reply Routes/index.js]: fixed

File: Routes/index.js (hunk 2 above)
Comment (biswasarpita) [reply Routes/index.js]: resolved

**Review by amitroy — APPROVED** [review]
Should be fine. Go ahead and merge it with master

**Pull request merged**